homework/index.js

```diff
@@ -0,0 +1,31 @@
+const dropDown = document.getElementById('dropDown');
+
+const listArray = [];
+
+const promise = fetch(
+  'https://api.github.com/orgs/HackYourFuture/repos?per_page=100',
+);
+
+promise
+  .then(newData => newData.json())
+  .then((data) => {
+    let listItem;
+    for (let i = 0; i < data.length; i++) {
+      listItem = document.createElement('option');
+      listItem.text = data[i].name;
+      dropDown.add(listItem);
+      listArray[i] = data[i];
+    }
+  });
+
+function reload() {
+  listArray.forEach((i) => {
```

Comment:
You could consider using `filter` on the array here, instead of the `forEach` + `if`. Also, consider naming your variable more descriptively, like `repo` :).